uwblueprint/community-fridge-kw
# feat: admin email verification donation scheduling

## Brief description. What is this change? 

### [Send admin email when new schedule is created](https://www.notion.so/uwblueprintexecs/Send-Admin-Email-when-New-Schedule-is-Created-22534d3dba774c15a6523ad6dbe5c6bf)



## Implementation description. How did you make this change?
* added new schedulingService `sendAdminEmailVerificationAfterDonationScheduled`
* hardcoded test admin email to test
* styling is consistent with donor email 
<img width="603" alt="Screen Shot 2022-02-04 at 1 42 30 PM" src="https://user-images.githubusercontent.com/53016529/152585066-16455c92-1e32-4855-97b6-f9a72752d37b.png">

<img width="638" alt="Screen Shot 2022-02-04 at 1 42 38 PM" src="https://user-images.githubusercontent.com/53016529/152585085-29b6b8c4-7f51-4dab-abd7-31a6579c5c51.png">

<img width="258" alt="screenshot of mobile view" src="https://user-images.githubusercontent.com/53016529/152585191-c575b4a8-f8c4-4689-94b0-92347ef28009.PNG">




## Steps to test
1. change hardcoded admin email within `sendAdminEmailVerificationAfterDonationScheduled` in `schedulingService.ts` to your test email
2. make donations and check if email is sent / and formatting is correct


## Checklist
